### PR TITLE
Improve UX after all activities sorted

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
Simplify design when all activities are already sorted
- add message on unsorted page
- remove navigation (sorted/unsorted) from all of the pages

### CODE REVIEW

**Comment by jessstubenbord** [comment]
nice :santa:

File: test/common/page_objects/sorted-activities-page.js [hunk 1]
```diff
@@ -16,6 +17,9 @@ const SortedActivitiesPage = function SortedActivitiesPage(browser) {
   this.clickReSortActivitiesLink = () => browser.click('[data-test="re-sort-link"]');
   this.getCategorySummary = (category) =>
     browser.text(`[data-test="category-${category}"] [data-test="show-hide-text"]`);
+  this.expectAt = accountId =>
+    expect(this.browserPath()).to.equal(`/${accountId}/activities/sorted`);
+  this.isNavigationDisplayed = () => !!browser.query('[data-test-id="navigation"]');
```
Comment (nathancashmoredwp) [inline test/common/page_objects/sorted-activities-page.js]: !! ... whats this for ?

**Comment by nathancashmoredwp** [comment]
👍 ... need to chat about !! as I think i could have used this in a couple of places if my understanding is correct.

File: test/common/page_objects/sorted-activities-page.js (hunk 1 above)
Comment (lukasz-kaniowski) [inline test/common/page_objects/sorted-activities-page.js]: double negative. 
i.e. 
```
$ node
> var a = 'string'
undefined
> a
'string'
> !!a
true
```

**Pull request merged**